cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -66,9 +73,28 @@ def test_appliance_scsi_control_vmware():
         casecomponent: Appliance
         caseimportance: critical
         initialEstimate: 1/4h
-    #TODO: yet to test this, once done, I will add steps. Test was not written by me originally.
     """
-    pass
+    url = (conf.cfme_data['basic_info']['cfme_images_url']['cfme_paravirtual_url_format'])\
+        .format(baseurl=conf.cfme_data['basic_info']['cfme_images_url']['baseurl'],
+            series=appliance.version.series(), ver=appliance.version)
+    logger.info("Downloading ova file for parvirtual vsphere scsi controller test from %s" % url)
```

Comment:
The value to format should be passed as an argument, not using the `%` syntax.

`logger.info('message: %s', url)`